prep-css-box-model/style.css

```diff
@@ -0,0 +1,15 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border-width: 3px;
+  border-style: double;
+  border-color: darkgray;
```

Comment:
You can combine this into a single CSS rule: `border: 3px double darkgrey;`

Reply:
this is a great tip thanks!